.travis.yml

```diff
@@ -1,16 +1,24 @@
 language: python
-python:
-  - "2.6"
-  - "2.7"
-  - "3.4"
-  - "3.5"
+matrix:
+  include:
+    - python: "2.6"
+      env: TOXENV=py26
+    - python: "2.7"
+      env: TOXENV=py27
+    - python: "3.4"
+      env: TOXENV=py34
+    - python: "3.5"
+      env: TOXENV=py35
 # command to install dependencies
 install:
   - "pip install -r requirements.txt"
```

Comment:
In theory setup.py would pull in the requirements from requirements.txt, and tox uses setup.py to install the package. This particular setup.py doesn't do that... in fact requirements.txt is empty.

Wonder if we should remove `- "pip install -r requirements.txt"` from this file and remove the requirements.txt file altogether, to make that clearer.